src/fallback.ts

```diff
@@ -400,7 +401,10 @@ export class GrpcClient {
           const decodedRequest = method.resolvedRequestType.decode(requestData);
           const requestJSON = method.resolvedRequestType.toObject(
             // TODO: use toJSON instead of toObject
-            decodedRequest
+            decodedRequest,
+            {
+              longs: String,
```

Comment:
can we add `enums: String` here too? This will make sense.

Reply:
I hesitated yesterday. I will add here. 
BTW, I didn't test this lines. No need to test this request conversion, it covered by `Message.toObject`; This expect to replace by use `toJSON` soon.